app/src/main/java/org/oppia/android/app/customview/SegmentedCircularProgressView.kt

```diff
@@ -66,15 +74,22 @@ class SegmentedCircularProgressView : View {
     chapterFinishedArcPaint.color =
       ContextCompat.getColor(context, R.color.oppiaProgressChapterFinished)
 
-    chapterNotFinishedArcPaint = Paint(Paint.ANTI_ALIAS_FLAG)
-    chapterNotFinishedArcPaint.style = Paint.Style.STROKE
-    chapterNotFinishedArcPaint.strokeCap = Paint.Cap.ROUND
-    chapterNotFinishedArcPaint.strokeWidth = strokeWidth
+    chapterInProgressArcPaint = Paint(Paint.ANTI_ALIAS_FLAG)
+    chapterInProgressArcPaint.style = Paint.Style.STROKE
+    chapterInProgressArcPaint.strokeCap = Paint.Cap.ROUND
+    chapterInProgressArcPaint.strokeWidth = strokeWidth
+    chapterInProgressArcPaint.color =
+      ContextCompat.getColor(context, R.color.oppiaProgressChapterInProgress)
+
+    chapterNotStartedArcPaint = Paint(Paint.ANTI_ALIAS_FLAG)
```

Comment:
Can we move it to helper function since this is being written here twice?

Reply:
This makes it consistent with `chapterFinishedArcPaint`. Doing these two with a helper function and `chapterFinishedArcPaint` without a helper function might be slightly misleading.

Reply:
WDYT?

Reply:
Since that is also using a similar process i would suggest moving both of them to a helper function.

Reply:
done